modules/systemd.nix

```diff
@@ -52,7 +52,8 @@ let
   unitType = unitKind:
     with types;
     let primitive = either bool (either int str);
-    in attrsOf (attrsOf (attrsOf (either primitive (listOf primitive)))) // {
+    in attrsOf
+    (attrsOf (attrsOf (oneOf [ primitive package (listOf primitive) ]))) // {
```

Comment:
I don't like `package` and don't think it's needed here. What I'd do is simply make `primitive = oneOf [ bool int str path ]` and leave this line unchanged.

Reply:
Fair enough!
I actually didn't notice that primitive was defined just above.